src/test/java/org/reaktivity/specification/amqp/streams/LinkIT.java

```diff
@@ -63,8 +63,8 @@ public void shouldExchangeAttachAsSender() throws Exception
 
     @Test
     @Specification({
-        "${scripts}/attach.sender.and.receiver/client",
-        "${scripts}/attach.sender.and.receiver/server"})
+        "${scripts}/attach.as.sender.and.receiver/client",
+        "${scripts}/attach.as.sender.and.receiver/server"})
```

Comment:
Suggest renaming `attach.as.sender.and.receiver` to `attach.as.sender.then.receiver`.
Also rename method to `shouldExchangeAttachAsSenderThenReceiver`